doc/api/dns.md

```diff
@@ -174,9 +174,10 @@ Valid values for `rrtype` are:
  * `'NAPTR'` - name authority pointer record
 
 The `callback` function has arguments `(err, addresses)`. When successful,
-`addresses` will be an array. The type of each  item in `addresses` is
-determined by the record type, and described in the documentation for the
-corresponding lookup methods.
+`addresses` will be an array. If resolving an SOA record, it will return an
+object with the same structure as is returned by the dns.resolveSoa method.
```

Comment:
Also, shouldn't it mention what would be the returned values if it is not an SOA record?